dashboard/src/components/TestsTable/IndividualTestsTable.tsx

```diff
@@ -34,12 +45,18 @@ const TableRowComponent = ({
   return (
     <TableRow key={row.id} className="border-b-0 hover:bg-lightBlue">
       {row.getVisibleCells().map(cell => {
-        return (
-          <TableCell key={cell.id}>
-            <Link {...linkProps}>
-              {flexRender(cell.column.columnDef.cell, cell.getContext())}
-            </Link>
+        return cell.column.id === 'status' ? (
+          <TableCell
+            key={cell.id}
+            className="cursor-pointer"
+            onClick={() => openLogSheet(rowIdx)}
```

Comment:
`useCallback` , you will probably need another component here that receives `index` as prop

Reply:
why I need another component?